allenact_plugins/manipulathor_plugin/manipulathor_constants.py

```diff
@@ -0,0 +1,116 @@
+"""Constant values and hyperparameters that are used by the environment."""
+import ai2thor
+import ai2thor.fifo_server
+from allenact_plugins.ithor_plugin.ithor_environment import IThorEnvironment
+
+MOVE_THR = 0.01
```

Comment:
I wasn't sure what this was, can you change it to `MIN_MOVE_THRESHOLD` or similar?